server/methods/search/requestMovie.js

```diff
@@ -62,7 +62,22 @@ Meteor.methods({
 			}
 		}
 
-		if (settings.movieApproval) {
+        
+		function approvedUser(user) {
```

Comment:
Should see if there's somewhere we can put this method so it can be used in both places. Don't want to make changes in one area but then not the other.